scripts/update_config.js

```diff
@@ -0,0 +1,157 @@
+/*
+ * LiskHQ/lisk-scripts/updateConfig.js
+ * Copyright (C) 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ * Usage Example:
+ * 		node scripts/update_config.js ../lisk-backup/config.json ./config.json
+ */
+
+const fs = require('fs');
+const readline = require('readline');
+const program = require('commander');
+const extend = require('extend');
+const lisk = require('lisk-elements');
+
+let oldConfigPath;
+let newConfigPath;
+let oldConfig;
+let newConfig;
+
+program
+	.version('0.1.1')
+	.option(
+		'-p, --password <string>',
+		"Password for secret encryption. This feature is only for testing purpose, don't use is it production.",
+		''
+	)
+	.arguments('<old_config_file> <new_config_file>')
+	.action((oldConf, newConf) => {
+		oldConfigPath = oldConf;
+		newConfigPath = newConf;
+
+		if (!oldConfigPath && !newConfigPath) {
```

Comment:
This check doesn't seem to run. :( https://github.com/tj/commander.js/issues/729

Reply:
Updated it, but if you put this check outside the action an check the variables value which were set inside action, then what's was the reason of doing all refactoring before.